src/main/webapp/resources/js/contexts/AnalysisSamplesContext.js

```diff
@@ -24,7 +24,7 @@ function AnalysisSamplesProvider(props) {
   const { analysisContext } = useContext(AnalysisContext);
   const [sampleDisplayHeight, setSampleDisplayHeight] = useState(null);
```

Comment:
Look at the effect called [useLayoutEffect](https://reactjs.org/docs/hooks-reference.html#uselayouteffect), I think it might work better here.

Reply:
Updated in 3675bbffdcf974bc6a42edaf1564e06d4973f442